x/auth/tx/mode_handler.go

```diff
@@ -12,6 +12,7 @@ var DefaultSignModes = []signingtypes.SignMode{
 	signingtypes.SignMode_SIGN_MODE_DIRECT,
 	signingtypes.SignMode_SIGN_MODE_DIRECT_AUX,
 	signingtypes.SignMode_SIGN_MODE_LEGACY_AMINO_JSON,
+	signingtypes.SignMode_SIGN_MODE_EIP_191,
```

Comment:
Adding this line would make the change state-machine breaking I think (for apps that use the SDK's default sign mode handler, i.e. 99% of them I guess).

Adding the proto enum entry seems fine to backport to 0.45, but I would remove this line here. Then, chains that want this can create their own `makeSignModeHandler` with this additional sign mode.

Reply:
removed

Reply:
@fedekunze this change is fine on master and should be there, amuary meant to remove on the backport. Do you want to add it back?